src/webgl/p5.Geometry.js

```diff
@@ -87,7 +87,59 @@ p5.Geometry = class Geometry {
 
     this.dirtyFlags = {};
   }
-
+  /**
+   * Removes the internal Colors of p5.Geometry
+   *
+   * @method clearColors
+   *
+   * @example
+   * <div>
+   * <code>
+   * let shape01;
+   * let shape02;
+   * let points = [];
+   *
+   * function setup() {
+   *   renderer = createCanvas(600, 600, WEBGL);
```

Comment:
I think we can take out the `renderer =` because we don't use it elsewhere.

Reply:
Oh yess... I will take out that.